cypress/e2e/volunteer.cy.js

```diff
@@ -0,0 +1,77 @@
+/// <reference types="cypress" />
+
+describe("Volunteer User Stories", () => {
+  // user navigates to main screen
+  beforeEach(() => {
+    cy.visit("http://localhost:3000/");
+  });
+
+  // user navigates to get-involved page via navbar
+  it("User can navigate to 'Get Involved' page from homewpage", async () => {
+    // user must click on hamburger menu to open navbar before navigating to get-involved page
+    cy.get('.MuiDrawer-root > .MuiPaper-root').should('be.not.visible');
+    cy.get('.hamburger-btn').click();
+    cy.get('.MuiDrawer-root > .MuiPaper-root').should('be.visible');
+
+    cy.get('.MuiDrawer-root > .MuiPaper-root > > ul > li > a').contains("Get Involved").click();
+    cy.location('pathname', {timeout: 5000}).should('include', '/get-involved');
+    cy.get('[data-testid="page-heading"]').contains("Get Involved");
+  });
+
+  
+  // user can use the page footer to navigate to get involved page
```

Comment:
Please no commented out code. If we aren't using it, should be deleted. Otherwise this is confusing to others reading later.. why was this here?